packages/core/components/DataTable/DataTable.tsx

```diff
@@ -250,9 +250,9 @@ const DataTable = (props: DataTableProps) => {
         {!config.table.showDownloadLinkBelow && <TableMediaControls />}
         <section
           id={tabbingId.replace('#', '')}
-          className={`data-table-container ${viewport} ${
-            !config.table.showDownloadLinkBelow ? 'download-link-above' : ''
-          }`}
+          className={`data-table-container  ${
```

Comment:
Use a helper function here since the class names are getting long and using ternaries.